### ISSUE TITLE
[BUG] Missing close icon on fab-sheet

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#2402]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
There is no close icon in the fab-sheet-button after it is opened.  

## Describe how to reproduce the bug 

1. Use the `kirby-fab-sheet` in a project that does not have ionicons installed

html:
``` html
<kirby-page title="Hello, world. This is Kirby" >
    <kirby-page-content>
        <kirby-fab-sheet horizontalAlignment="right">
            <kirby-icon name="write-message"></kirby-icon>
            <kirby-action-sheet
              header="Your action sheet header"
              subheader="Your action sheet subheader"
              [items]="items"
            >
            </kirby-action-sheet>
          </kirby-fab-sheet>
    </kirby-page-content>
</kirby-page>
```

ts:
```ts
 items: ActionSheetItem[] = [
    { id: '1', text: 'Option 1' },
    { id: '2', text: 'Option 2' },
    { id: '3', text: 'Option 3' },
  ];


  showActionsheet() {
    const config: ActionSheetConfig = {
      header: 'Your action sheet header',
      subheader: 'Your action sheet sub-header',
      items: [
          { id: '1', text: 'Option 1' },
          { id: '2', text: 'Option 2' },
          { id: '3', text: 'Option 3' },
      ],
      cancelButtonText: 'Custom cancel',
    };

    this.modalController.showActionSheet(config);
  }
```

2. Click on the fab sheet button
3. See that there is no close icon present in the button (backdrop is missing on picture below as well, but that is solved in a separate issue - #2401): 
![image](https://user-images.githubusercontent.com/42470636/182372974-d2e6d017-7a39-4ff4-8177-85d53113caa3.png)

## Which Kirby version was used?
v6.1.1

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
